backend.native/tests/objcexport/values.swift

```diff
@@ -1278,6 +1278,9 @@ func testGH3825() throws {
     try assertEquals(actual: count, expected: 3)
 }
 
+func testKt35940() throws {
```

Comment:
Please move this test to a separate file.
See `backend.native/tests/objcexport/gh4002.*` as an example.